integration-tests/pom.xml

```diff
@@ -190,6 +195,7 @@
                                 -Dfile.encoding=UTF-8
                                 -Dtestrunfactory=org.testng.DruidTestRunnerFactory
                                 -Ddruid.test.config.dockerIp=${env.DOCKER_IP}
+                                -Ddruid.test.config.hadoopDir=${env.HADOOP_DIR}
```

Comment:
The ${env.xxx} things have to be present in environment variables when the mvn command is given to run tests with docker.   Magically, the values from the environment are injected into DockerConfigProvider.
 
If we had docker set up so it could do hadoop, we'd edit integration-tests/README.md to say to export HADOOP_DIR to the name of a directory tonaining batchHadoop1, where the test data is in hdfs.  

Even though the tests can't be run in docker yet, I wanted to add this line and make the mods to DockerConfigPhrovider.java to be able to do the configuration for the hdfs location, while I had it in my head what needed to be done.  Maybe I should file an issue about hadoop-in-docker, and say that the support for using it is already there in the code, and what would need to be added to the README.